features/gift_exchanges/challenge_giftexchange.feature

```diff
@@ -48,6 +48,18 @@ Feature: Gift Exchange Challenge
       And I follow "Gift Exchange Challenges"
     Then I should see "My Gift Exchange"
 
+  Scenario: Gift exchange in closed collection appears in list of open challenges
+    Given I am logged in as "mod1"
+      And I have created the gift exchange "My Gift Exchange"
+      And I am on "My Gift Exchange" gift exchange edit page
+    When I check "Sign-up open?"
+      And I submit
+    When I am on "My Gift Exchange" collection edit page
```

Comment:
For style it would be nice if these `When` were `And` instead so we don't repeat `When` without a `Then` afterwards